lsquic.nimble

```diff
@@ -45,19 +47,17 @@ before install:
     exec "nasm -f win64 ./libs/boringssl/gen/crypto/chacha-x86_64-win.asm -o chacha-x86_64-win.o"
 
 task format, "Format nim code using nph":
-  exec "nimble install nph"
```

Comment:
do not waist time on `nimble install`.
better do it manually if `exec "nph"` fails